### PR TITLE
entrypoint permission issue

### PR DESCRIPTION
**Pull request by ronniedada**
docker: Error response from daemon: invalid header field value "oci runtime error: container_linux.go:247: starting container process caused \"exec: \\\"/entrypoint.sh\\\": permission denied\"\n".

### CODE REVIEW

File: Dockerfile-devenv [hunk 1]
```diff
@@ -20,5 +20,6 @@ RUN apt-get install -y apt-transport-https ca-certificates gnupg2 \
     && echo "deb https://apt.dockerproject.org/repo debian-jessie main" > /etc/apt/sources.list.d/docker.list \
     && apt-get update && apt-get install -y docker-engine
 COPY scripts/devenv-entrypoint.sh /entrypoint.sh
+RUN chmod +x /entrypoint.sh
```
Comment (declanshanaghy) [inline Dockerfile-devenv]: The checked in file should be marked executable then there would be no need to do this extra step in the dockerfile.

AFAIK it already is executable, no?

Comment (coccyx) [inline Dockerfile-devenv]: I fixed the executable bit a few merges ago, or at least I thought

Comment (ronniedada) [inline Dockerfile-devenv]: Should be the obsolete docker image at that time.  Pulled with the today's, looks good.

**Comment by ronniedada** [comment]
working, close now.

**Pull request closed without merging**